src/index.html

```diff
@@ -9,10 +9,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is linking to a `.scss` file (`styles/main.scss`). Browsers cannot directly interpret SCSS files. You should link to the compiled CSS file instead, such as `styles/main.css`.